src/core/componentSystems/PhysicsSystem.js

```diff
@@ -267,7 +269,11 @@ export class PhysicsSystem extends MRSystem {
     updateUIBody(entity) {
         this.tempBBox.setFromCenterAndSize(entity.object3D.position, new THREE.Vector3(entity.width, entity.height, 0.002));
 
-        this.tempWorldScale.setFromMatrixScale(entity.object3D.matrixWorld);
+        if (entity instanceof MRPanelEntity) {
+            this.tempWorldScale.setFromMatrixScale(entity.panel.matrixWorld);
+        } else {
+            this.tempWorldScale.setFromMatrixScale(entity.object3D.matrixWorld);
+        }
```

Comment:
it would be better to do this as

```
this.tempWorldScale.setFromMatrixScale( (entity instanceof MRPanelEntity) ? entity.panel.matrixWorld : entity.object3D.matrixWorld );
```